### PR TITLE
feat: set analyze v2 api to ProjectBilling widget

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
😇
스크린샷 2023-09-20 오전 9 18 07

### CODE REVIEW

**Review by WANZARGEN — APPROVED** [review]
I've primarily focused on reviewing the API changes. Thank you for your work!

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/services/project/project-detail/project-summary/modules/ProjectBilling.vue [hunk 1]
```diff
@@ -82,35 +82,32 @@
...
+                    <!--                            >-</span>-->
+                    <!--                        </template>-->
+                    <!--                        <span v-else-->
+                    <!--                              :key="field.name"-->
+                    <!--                              class="col-service"-->
+                    <!--                              :class="{'link-text': !!value.to.name }"-->
+                    <!--                        >-->
+                    <!--                            <span>{{ value.name }}</span>-->
+                    <!--                        </span>-->
+                    <!--                    </template>-->
```
Comment (piggggggggy) [inline apps/web/src/services/project/project-detail/project-summary/modules/ProjectBilling.vue:110]: question: Why did you leave it as annotation?

File: apps/web/src/services/project/project-detail/project-summary/modules/ProjectBilling.vue [hunk 2]
```diff
@@ -238,10 +244,53 @@ export default {
...
+                    // const dateFormat = state.selectedDateType === DATE_TYPE.monthly ? 'YYYY-MM' : 'YYYY-MM-DD';
+                    // const pastDate = dayjs.utc(d.date).subtract(1, timeUnit).format(dateFormat);
+                    // const pastCost = results.find((bd) => bd.date === pastDate)?.cost_sum || 0;
+                    // costData[d.date] = {
+                    //     cost: currencyMoneyFormatter(d.value, state.currency),
+                    // };
+                    // if (pastCost && pastCost < d.cost_sum && (d.cost_sum - pastCost) / Math.abs(pastCost) > 0.5) {
+                    //     costData[d.date].color = 'red';
+                    // }
+                    d.cost_sum.forEach((cost) => {
```
Comment (piggggggggy) [inline apps/web/src/services/project/project-detail/project-summary/modules/ProjectBilling.vue:283]: Same here~

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Pull request merged**